pkgs/development/compilers/ldc/bootstrap.nix

```diff
@@ -0,0 +1,10 @@
+{ callPackage }:
+callPackage ./binary.nix {
+  version = "1.19.0";
```

Comment:
Was there a specific reason why the previous version is used for bootstrapping?
I can think of no technical reason but it would make it even easier if the same version is used for bootstrapping.
This way only one version specifier needs to exist.